src/components/homes-feed/homes-feed.tsx

```diff
@@ -19,42 +19,21 @@ const Item = styled(Paper)(({ theme }) => ({
 }));
 
 const HomesDisplay: FC = () => {
-  const [data, setData] = useState<HomeApiResponse | null>(null);
-
-  useEffect(() => {
-    fetchPhotos('house')
-      .then(result => {
-        setData(result);
-      })
-      .catch(() => {
-        console.error('something went wrong!');
-      });
-  }, []);
-
-  if (data === null) {
-    return <div>Loading...</div>;
-  } else if (data?.errors) {
-    return (
-      <div>
-        <div>{data.errors[0]}</div>
-        <div>PS: Make sure to set your access token!</div>
-      </div>
-    );
-  } else {
-    return (
-      <Box sx={{ flexGrow: 1 }}>
-        <Grid container spacing={2}>
-          {data?.response.results.map((photo: Photo) => (
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <Grid container spacing={2}>
+        {Array.from(homesMockData).map(
```

Comment:
Try removing Array.from since the mock data is an array (no need to convert to a shallow-copied array instance).